test/tools/isoltest.cpp

```diff
@@ -79,9 +79,18 @@ class TestFilter
 		m_filterExpression = regex{"(" + filter + "(\\.sol|\\.yul))"};
 	}
 
-	bool matches(string const& _name) const
+	bool matches(fs::path const& _path, string const& _name) const
 	{
-		return regex_match(_name, m_filterExpression);
+		if (regex_match(_name, m_filterExpression))
+		{
+			stringstream stream;
+			ifstream inputStream(_path.string());
```

Comment:
Wouldn't using `util::readFileAsString` from libsolutil be better?

Reply:
much better!